default_warehouse_from_sale_team/__openerp__.py

```diff
@@ -12,6 +12,7 @@
     'version': '8.0.2.0.0',
     'depends': [
         'sale_stock',
+        'delivery',
```

Comment:
The field  ```weight_uom_id``` is charged for default in this module, without ```delivery``` depends, a bug occurs ``` "null value in column "weight_uom_id" violates not-nu
ll constraint ```